histograms/models.py

```diff
@@ -39,7 +39,7 @@ class Meta:
 
 
 class RunHistogram(HistogramBase):
-    run = models.ForeignKey(Run, on_delete=models.CASCADE)
+    run = models.ForeignKey(Run, on_delete=models.CASCADE, related_name="run_histogram")
```

Comment:
Minor remark:

We could even drop the `run_` part and just leave `histograms`, since you access this from `Run` instances. The use case will be like follows:

```python3
from data_taking_objects.models import Run

r = Run.objects.all()
rh = r.histograms  # Related objects of a ForeignKey are multiple entries in the related table, hence the plural
 
```